doc/BuildProgramVisualStudio.md

```diff
@@ -0,0 +1,66 @@
+# Introduction #
+
+This is a first step in documenting what steps are needed to compile FreeRCT on Windows using Visual Studio. We hope to reduce the length of this list later on.
```

Comment:
Perhaps also point back to the 'linux' build file?

I also wonder whether the former should be renamed to indicate it's for unix/linux